### PR TITLE
#97 Stiffness per constraint type Docs

### PR DESCRIPTION
**Pull request by josevalverdeinbibo**
### Tasks

- Updated attribute tables adding the new stiffness override attributes.
- Added a note for the deprecated attributes and explaining the behaviour when the checkbox is enabled.
- Updated screenshots showing the "Advanced Settings" section.

Issue of this PR: #97

### CODE REVIEW

File: docs/maya/muscle.md [hunk 1]
```diff
@@ -63,9 +63,24 @@ To create an AdnMuscle, follow these steps:
...
 
+#### Override Constraint Stiffness
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Solver Stiffness**         | Float |      | ✗ | Shows the global stiffness value currently used by the solver. |
+| **Attachments To Geometry**  | Float | -1.0 | ✓ | Sets the stiffness override value for attachment to geometry constraints. If the value is less than 0.0, the global stiffness will be used. Otherwise, this custom stiffness will override the global stiffness. |
+| **Attachments To Transform** | Float | -1.0 | ✓ | Sets the stiffness override value for attachment to transform constraints. If the value is less than 0.0, the global stiffness will be used. Otherwise, this custom stiffness will override the global stiffness. |
+| **Fiber Constraints**        | Float | -1.0 | ✓ | Sets the stiffness override value for fiber constraints. If the value is less than 0.0, the global stiffness will be used. Otherwise, this custom stiffness will override the global stiffness. |
```
Comment (lucasferryinbibo) [inline docs/maya/muscle.md:75]: Should we also add a note about the ranges for all stiffness values?

File: docs/maya/ribbon.md [hunk 2]
```diff
@@ -61,9 +61,23 @@ Follow this steps to create an AndRibbonMuscle deformer:
 | :--- | :--- | :------ | :--------- | :---------- |
 | **Use Custom Stiffness**                  | Boolean | False          | ✓ | Toggles the use of a custom stiffness value. If custom stiffness is used, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead. |
 | **Stiffness**                             | Float   | 10<sup>5</sup> | ✓ | Sets the custom stiffness value. Its value must be greater than 0.0. |
-| **Override Shape Preserve Stiffness** | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |
+| **Override Shape Preserve Stiffness**     | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |
```
Comment (lucasferryinbibo) [inline docs/maya/ribbon.md:64]: Can you also please check if there are any other screenshots that need to be updated?

**Review by lucasferryinbibo — CHANGES_REQUESTED** [review]
Some smalle changes. Please also add @carlosmonteagudoinbibo as a reviewer. I think we should leave this PR open so he's algned with the final changes done in the implementation if that sounds good to you. Thanks!

File: docs/maya/muscle.md [hunk 3]
```diff
@@ -63,9 +63,24 @@ To create an AdnMuscle, follow these steps:
...
+| **Shape Preservation**       | Float | -1.0 | ✓ | Sets the stiffness override value for shape preservation constraints. If the value is less than 0.0, the global stiffness will be used. Otherwise, this custom stiffness will override the global stiffness. |
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/maya/muscle.md:76]: As we are already mentioning "If the *Override Shape Preserve Stiffness* checkbox is enabled, the *Shape Preserve Stiffness* value will be used to override the stiffness for shape preservation constraints.", I think we can add an extra clarification to the *Shape Preservation* saying "This value is only considered if the *Override Shape Preserve Stiffness* checkbox is disabled.". Let me know if you disagree.

File: docs/maya/ribbon.md (hunk 2 above)
Comment (josevalverdeinbibo) [reply docs/maya/ribbon.md:64]: I checked all the screenshots that were showing the "Advanced Settings" section, anyway I will double check.

Comment (josevalverdeinbibo) [reply docs/maya/ribbon.md:64]: There are no other screenshots that need to be updated.

**Comment by josevalverdeinbibo** [comment]
We are leaving this PR open for now so that @carlosmonteagudoinbibo is aligned with the changes when he comes back.

File: docs/maya/muscle.md [hunk 4]
```diff
@@ -63,9 +63,24 @@ To create an AdnMuscle, follow these steps:
...
-| **Override Shape Preserve Stiffness** | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |
+| **Override Shape Preserve Stiffness**     | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |
 | **Shape Preserve Stiffness**              | Float   | 10<sup>3</sup> | ✓ | Sets the stiffness shape preservation override value. Its value must be greater than 0.0. |
 
+#### Override Constraint Stiffness
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Solver Stiffness**         | Float |      | ✗ | Shows the global stiffness value currently used by the solver. |
+| **Attachments To Geometry**  | Float | -1.0 | ✓ | Sets the stiffness override value for attachment to geometry constraints. If the value is less than 0.0, the global stiffness will be used. Otherwise, this custom stiffness will override the global stiffness. |
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/maya/muscle.md]: I finally added it as a note. I avoided referring to "projection" in case a user is not used to that concept.

**Pull request merged**